docs/templates/epacems_child.rst.jinja

```diff
@@ -56,18 +54,17 @@ A plain English explanation of the requirements of Part 75 is available in secti
 
 {% block original_data %}
 EPA CAMD publishes the CEMS data in an online `data portal <https://campd.epa.gov/>`__.
-The files are available in a prepackaged format, accessible via a `user interface <https://campd.epa.gov/data/custom-data-download>`__
-or `FTP site <https://gaftp.epa.gov/DMDnLoad/>`__ with each downloadable zip file
-encompassing a year of data.
+The files are available in a prepackaged format, accessible via a `user interface
+<https://campd.epa.gov/data/custom-data-download>`__.
 {% endblock %}
 
 {% block notable_irregularities %}
 
-CEMS is enourmous
------------------
-CEMS is by far the largest dataset in PUDL what with hourly records for
-thousands of plants spanning decades. For this reason, we distribute CEMS data using `Apache
-Parquet <https://parquet.apache.org/>`__ files rather than in the main PUDL database.
+CEMS is enormous
+----------------
+CEMS is by far the largest dataset in PUDL what with hourly records for thousands of
+plants spanning decades. For this reason, we distribute CEMS data using `Apache Parquet
+<https://parquet.apache.org/>`__ files rather than in the main PUDL database.
```

Comment:
I find this terminology confusing - it is part of our 'main database', just not part of the main PUDL SQLite file. Can we just say, "rather than using SQLite" here?

Reply:
Updated language.